SEPACreditTransfer.pas

```diff
@@ -83,6 +83,7 @@   TCreditTransferTransactionInformation = class
   TCreditTransferPaymentInformation = class
   private
     fPmtInfId: String;                                 // payment information identification
+    fBtchBookg:boolean;                                // Batchbooking parameter (at Sepa level default true if not defined)
```

Comment:
Please format the same as the other lines (whitespace before type, type with capital letter) - also in the other file, of course.